grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -603,7 +603,17 @@ class OccurrenceTagLib {
 
                         td(class: 'search-results-cell', *:properties) {
                             try {
-                                mkp.yieldUnescaped(formatValue(occurrence.get(column), column))
+                                def value = occurrence.get(column)
+
+                                if(column == 'scientificName') {
+                                    // Record view link.
+                                    a(
+                                        href: g.createLink(url:"${request.contextPath}/occurrences/${occurrence.uuid}"),
+                                        value ? value : g.message(code:'formatListRecordRow.viewRecord')
```

Comment:
space after ":" on both lines would be nice

Reply:
Ok, note to self: never simply copy anything that exists in that code.